src/frontend/web_application/src/components/MessageList/components/Message/style.scss

```diff
@@ -1,66 +1,123 @@
 @import '../../../../styles/vendor/bootstrap_foundation-sites';
 
 .m-message {
-  border-bottom: 1px solid $co-color__bg__back;
+  @include flex-grid-row($size: expand);
+  margin-bottom: $co-grid__gutter;
+  font-size: .9rem;
```

Comment:
yes! I asked @Lauth0 for font-size specs. btw I didn't find where font-sizes are declared. Are we using default foundation font-sizes?